modules/nf-core/cellranger/vdj/meta.yml

```diff
@@ -0,0 +1,43 @@
+name: cellranger_vdj
+description: Module to use Cell Ranger's pipelines analyze sequencing data produced from Chromium Single Cell Immune Profiling.
+keywords:
+  - align
+  - vdj
+  - reference
+  - immunoprofiling
+  - single-cell
+  - cellranger
+tools:
+  - cellranger:
+      description: Cell Ranger processes data from 10X Genomics Chromium kits. `cellranger vdj` takes FASTQ files from `cellranger mkfastq` or `bcl2fastq` for V(D)J libraries and performs sequence assembly and paired clonotype calling. It uses the Chromium cellular barcodes and UMIs to assemble V(D)J transcripts per cell. Clonotypes and CDR3 sequences are output as a `.vloupe` file which can be loaded into Loupe V(D)J Browser.
+      homepage: https://support.10xgenomics.com/single-cell-vdj/software/pipelines/latest/what-is-cell-ranger
+      documentation: https://support.10xgenomics.com/single-cell-vdj/software/pipelines/latest/tutorial/tutorial-vdj
+      tool_dev_url: https://support.10xgenomics.com/single-cell-vdj/software/pipelines/latest/tutorial/tutorial-vdj
+      licence: 10x Genomics EULA
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - reads:
+      type: file
+      description: |
+        List of input FastQ files of size 1 and 2 for single-end and paired-end data,
+        respectively.
```

Comment:
I see there is no if/else stratement based on meta for this. If there is a specific naming scheme taht must be  uysed for the tool to pick up the pairs I would add this here.

Reply:
added pattern descriptor to match [Cellranger expected FASTQ format](https://support.10xgenomics.com/single-cell-vdj/software/pipelines/latest/using/fastq-input):
```
${Sample_Name}_S1_L00${Lane_Number}_${I1,I2,R1,R2}_001.fastq.gz
```